src/common/components/repositories-home/index.js

```diff
@@ -119,10 +119,19 @@ class RepositoriesHome extends Component {
       );
     }
 
+    // switch over to snapcraft.io/<snap_name>/builds notification
+    const infoNotification = (
+      <Notification appearance='information'>
+        We are updating the way you build from GitHub to integrate better within
+        Snapcraft. <a href="#">Learn more</a>
```

Comment:
Maybe (once the URL is there) this could open in new window?

```suggestion
        Snapcraft. <a href="https://snapcraft.io/blog" target="_blank" rel="noopener noreferrer">Learn more</a>
```